Lib/idlelib/idle_test/run_coverage.py

```diff
@@ -0,0 +1,152 @@
+import argparse
+import os
+import pathlib
+import subprocess
+import sys
+import textwrap
+
+
+def main():
+    idlelib_path = find_idlelib_path()
+    sources_root_path = idlelib_path.parent.parent
+    venv_path, venv_python_path = ensure_venv(sources_root_path)
+
+    module_names = sorted(get_module_names(idlelib_path))
+
+    parser = argparse.ArgumentParser(
+        description='Run tests for an idlelib module '
+                    'and collect coverage stats',
+    )
+    parser.add_argument('module', choices=['all'] + module_names)
```

Comment:
We should add a flag so we don't have to create and launch an HTML report every time (quite often the stdout report will be sufficient - though the HTML report should be on by default).
`parser.add_argument('-H', '--no-html', action='store_true', help='Do not create and launch an HTML coverage report')`

Reply:
Great idea :+1: